app/assets/javascripts/dashboard/protocols.js.coffee

```diff
@@ -55,6 +57,13 @@ $(document).ready ->
         $.ajax
           type: 'get'
           url: "/dashboard/protocols/#{protocol_id}/display_requests"
+          success: (data) ->
```

Comment:
Send the request, get the modal back through JSON and open it, then enable the bootstraptables and enable the selectpickers